Makefile.in

```diff
@@ -62,7 +63,7 @@ ifeq "$(enable_checks)" "true"
  # Debug + checks
  OPT_FLAGS=-g
  ADA_FLAGS=-gnata
- LIB_CFLAGS=
+ LIB_CFLAGS= --AS=$(AS)
```

Comment:
I don't understand why you need that.
It looks like at some point you are using the native ghdl with a cross assembler.  See the full discussion.

Reply:
This is required to call the correct (cross) assembler to build the libraries, see make rule https://github.com/hackfin/ghdl-cross.mk/blob/671ea4348fdd7b52a8d7e44ac607e798d4f65d2d/recipes/ghdl-cross.mk#L105

I don't know yet how this could be automated, do you think it would make sense to look at integration of some make auxiliaries into ghdl? Would be nice to have a ' configure' one-liner, but also introduces more complexity.

Reply:
I see a problem: the user also have to specify --AS=xxx when invoking ghdl, doesn't he ?

Reply:
Yes, for now that's the case. I believe it would make sense to add default path strings in default_paths.ads.in. Same would apply to the `--LINK ` option. I am in (slow) process of making a few packages for testing (debian), once that's somewhat stable, I'll likely push another commit.

Reply:
For former is for ghdl1-gcc, the latter for the ghdl driver.

Reply:
I've added some yet unstable debian/ rules to ghdl-cross.mk that install the target specific libs for ghdl in /usr/$(TARGET_ARCH)/lib/ghdl, when cross-compiling. Likewise, 7d95c56 adds the specific Lib_Prefix for multiple target GHDL instances. Since this is all still gcc specific, it's done for `./gcc/vhdl/default_paths.ads` only.

I don't know who is currently maintaining the debian packages, but I'm happy to assist whoever who wishes to integrate that into an official repo and make it a source package, once 0.36 is frozen for release.